tensorflow/tools/lib_package/BUILD

```diff
@@ -182,6 +183,7 @@ genrule(
         "@zlib_archive//:zlib.h",
     ] + if_mkl([
         "//third_party/mkl:LICENSE",
+        "@mkl_dnn//:LICENSE",
```

Comment:
Should it be `"//third_party/mkl_dnn:LICENSE"`?

Reply:
but I can try "//third_party/mkl_dnn:LICENSE" and see.